Database-Features/no-default-driver-registered-not-written-in-the-default-jdbc-driver-path.md

```diff
@@ -0,0 +1,66 @@
+# No default DRIVER registered (not written in the Default JDBC Driver Path)
+
+## Problem
+
+We have stored the driver in **bucketfsfordrivers/mydrivers/jdbc**. However, the connection does not work and the following error message appears:
```

Comment:
If "bucketfsfordrivers/mydrivers/jdbc" is a BucketFS path, then it should start with "/buckets".